src/Analysis/GWASWizard/GWASQuantitative.jsx

```diff
@@ -83,14 +84,20 @@ const GWASQuantitative = ({ resetGWASType, refreshWorkflows }) => {
 
 
   const generateStep = () => {
+    let stepInfo = {
+      step: current,
+      workflowName: "quantitative"
+    }
```

Comment:
this would be a good place to include an additional field to parametrize "Open Tour" text on the button i think